code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -12,7 +12,7 @@
 	icon = 'icons/obj/machines/shuttle_manipulator.dmi'
 	icon_state = "holograph_on"
 	/// Used for cooldown, very obvious name, required due to shuttles spawning in the same location and causing the server to implode
-	var/shuttle_and_preview_cooldown = 0
+	var/shuttle_and_preview_cooldown = 0  // TODO MAKE SURE THIS NEW BEHAVIOR IS PRESENT
```

Comment:
Is this done? Or a TODO for a PR in the future?

Reply:
Good catch. I need to verify that the manipulator still has the delay that it had before.